samples/subsys/fs/littlefs/boards/nucleo_f429zi.overlay

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020 Linaro Limited.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* LittleFS expects at least 2 flash sectors with a constant erase size. */
+/* For STM32F2/F4/F7 series, it means that at least 2 sectors of the same */
+/* size should be made available for storage partition. */
+
+/* Applied to stm32f429zi used without bootloader, the only solution is */
+/* to allocate the 2 consecutive 256Kb sectors. */
+/* To make it working on nucleo_f429zi, it means that default partitions */
+/* should be arranged as follows: */
+
+/* Delete existing partitions that needs to be modified */
+/delete-node/ &storage_partition;
+/delete-node/ &slot0_partition;
+/delete-node/ &slot1_partition;
+
+&flash0 {
+	partitions {
+
+		/* This partition is only needed to satisfy main file */
+		/* chosen { zephyr,code-partition = &slot0_partition; } */
+		slot0_partition: partition@10000 {
+			label = "image-0";
+			reg = <0x00010000 0x00010000>;
+		};
+
+		/* storage image slot: 2 x 256KB sectors */
```

Comment:
How much testing has this gotten?  IIRC littlefs does expect to be able to allocate file system data on a per-block basis; the block size is extracted from the flash map.  Two blocks will not allow creation of a filesystem.

Reply:
cf https://github.com/zephyrproject-rtos/zephyr/pull/28387#issuecomment-692675878